common/handlers/users/create.go

```diff
@@ -0,0 +1,72 @@
+package users
+
+import (
+	"context"
+
+	dsw "github.com/aserto-dev/go-directory/aserto/directory/writer/v3"
+	"github.com/aserto-dev/scim/common/convert"
+	"github.com/aserto-dev/scim/common/model"
+	"github.com/elimity-com/scim"
+	serrors "github.com/elimity-com/scim/errors"
+)
+
+func (u UsersResourceHandler) Create(ctx context.Context, attributes scim.ResourceAttributes) (scim.Resource, error) {
```

Comment:
This is a really long function. I think it can be broken into several smaller ones.